cogs/StaffAppsBackoffice.py

```diff
@@ -0,0 +1,966 @@
+            field_title = "Application ID, Discord Name, Original Submission Time"
+            field_content = ""
+            for row in data[:10]:
+                field_content += (
+                    f"{row[0]}, {row[1]}, {datetime.fromtimestamp(int(row[2]))}\n"
+                )
+            fields = [[field_title, field_content, False]]
+            embed = EmbedBuilder(
+                title=title,
+                description="",
+                fields=fields,
+                color=0x9D0006,
+            )  # soft red
+            if len(data) > 10:
+                await ctx.respond(
+                    embed=embed.build(),
+                    view=staffAppsSeeDeniedAccepted(ctx.author, data, "d"),
+                    ephemeral=True,
+                )
+            else:
+                await ctx.respond(embed=embed.build(), ephemeral=True)
+            Log(f"{username} viewed denied applications")
+            return
+
+        elif subcommand == "accepted":
+            data = self.cursor.execute(
+                "select appId, discordName, submissionTime from application where status = 9",
+            ).fetchall()
+            title = f"Accepted Applications page 1/{len(data)//10+1}"
+            field_title = "Application ID, Discord Name, Original Submission Time"
+            field_content = ""
+            for row in data[:10]:
+                field_content += (
+                    f"{row[0]}, {row[1]}, {datetime.fromtimestamp(int(row[2]))}\n"
+                )
+            fields = [[field_title, field_content, False]]
+            embed = EmbedBuilder(
+                title=title,
+                description="",
+                fields=fields,
+                color=0x80C545,
+            )  # hwh green
+            if len(data) > 10:
+                await ctx.respond(
+                    embed=embed.build(),
+                    view=staffAppsSeeDeniedAccepted(ctx.author, data, "a"),
+                    ephemeral=True,
+                )  #
+            else:
+                await ctx.respond(embed=embed.build(), ephemeral=True)
+            Log(f"{username} viewed accepted applications")
+            return
+
+        elif subcommand == "spam":
+            data = self.cursor.execute(
+                "select * from user where markedSpam = 1",
+            ).fetchall()
+
+            if len(data) == 0:
+                await ctx.respond(
+                    "No users are currently marked as spam.",
+                    ephemeral=True,
+                )
+                return
+
+            first_user_data = self.cursor.execute(
+                "select * from application where uid = ?",
+                (data[0][0],),
+            ).fetchall()
+            count = len(first_user_data)
+            name = first_user_data[0][3]
+            most_recent = datetime.fromtimestamp(
+                max([int(data[10]) for data in first_user_data]),
```

Comment:
Replace unnecessary comprehension with a generator
```suggestion
                max(int(data[10]) for data in first_user_data),
```